### PR TITLE
Sonar: This block of commented-out lines of code should be removed

### PR DESCRIPTION
**Pull request by qmonmert**
Should remove a lot of Sonar code smells like https://sonarcloud.io/project/issues?impactSoftwareQualities=MAINTAINABILITY&resolved=false&id=jhipster-sample-application&open=AY92SBeNzvwT9Xfig0tw

Related to #25231 

---

Please make sure the below checklist is followed for Pull Requests.

- [ ] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
- [ ] Tests are added where necessary
- [ ] The JDL part is updated if necessary
- [ ] [jhipster-online](https://github.com/jhipster/jhipster-online) is updated if necessary
- [ ] Documentation is added/updated where necessary
- [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/main/CONTRIBUTING.md) are followed

When you are still working on the PR, consider converting it to Draft (below reviewers) and adding `skip-ci` label, you can still see CI build result at your branch.

### CODE REVIEW

File: generators/gatling/templates/src/test/java/gatling/simulations/_entityClass_GatlingTest.java.ejs [hunk 1]
```diff
@@ -41,6 +41,7 @@ import org.slf4j.LoggerFactory;
 /**
  * Performance test for the <%= entityClass %> entity.
  */
+@SuppressWarnings("java:S125")
```
Comment (mshima) [inline generators/gatling/templates/src/test/java/gatling/simulations/_entityClass_GatlingTest.java.ejs:44]: I think we should remove the commented code.
I think we can do something like:

```suggestion
 *
 * @see <a href="https://github.com/jhipster/generator-jhipster/tree/v<%- jhipsterVersion %>/generators/gatling#logging-tips">Logging tips</a>
 */
```
Add a README.md in https://github.com/jhipster/generator-jhipster/tree/main/generators/gatling with:
````
## Logging tips

```
LoggerContext context = (LoggerContext) LoggerFactory.getILoggerFactory();
{
    // Log all HTTP requests
    //context.getLogger("io.gatling.http").setLevel(Level.valueOf("TRACE"));
    // Log failed HTTP requests
    //context.getLogger("io.gatling.http").setLevel(Level.valueOf("DEBUG"));
}
```
````

Comment (qmonmert) [reply generators/gatling/templates/src/test/java/gatling/simulations/_entityClass_GatlingTest.java.ejs:44]: @mshima ok let's try this

File: generators/gatling/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+## Logging tips
```
Comment (mshima) [inline generators/gatling/README.md:1]: Add a little description to the generator.
```suggestion
# Gatling sub-generator

Generates load testing tool for entities using [Gatling](https://gatling.io/).

## Logging tips
```

File: generators/gatling/README.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Gatling sub-generator
+
+Generates load testing tool for entities using [Gatling](https://gatling.io/).
```
Comment (mraible) [inline generators/gatling/README.md:3]:
```suggestion
Generates load testing simulations for entities using [Gatling](https://gatling.io/).
```

**Review by mraible — DISMISSED** [review]

**Review by mshima — APPROVED** [review]

**Review by mraible — APPROVED** [review]

**Review by DanielFran — APPROVED** [review]

**Pull request merged**